engines/query-sparql/test/QuerySparql-test.ts

```diff
@@ -446,6 +446,38 @@ describe('System test: QuerySparql', () => {
     });
   });
 
+  // TODO: Re-enable this once https://github.com/RubenVerborgh/AsyncIterator/pull/45 is closed
```

Comment:
I'm not a big fan of commented code :P
Could we move this to an issue (if it still is a problem)?